.changeset/plenty-moose-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/tokens": minor
```

Comment:
Alternatively, I could see this being a `major`. It has the potential to cause visual changes and introduce some layout shifts that people might not be expecting.

Reply:
Overall this PR looks solid! Great work on this. I can see the difference here:

### Prod URL
<img width="311" alt="Screenshot 2024-09-27 at 9 14 37 PM" src="https://github.com/user-attachments/assets/ffd84572-bdd5-4ba9-9c48-4ac7a2cc9dae">

### Deploy URL
<img width="316" alt="Screenshot 2024-09-27 at 9 14 43 PM" src="https://github.com/user-attachments/assets/1cf37698-9eb4-4a86-8457-6cb8f88067c7">

This space feels a lot better visually 🤩

I would say this a `major` change because text in these components are going from two lines to one within a given horizontal space. People may or may not want adjust the width of dropzones or illustrated messages in their current layouts after the release.

Reply:
Ok! I made the change so that it's now a `major`. Thanks!